Python/test/test_magcoords.py

```diff
@@ -51,8 +50,8 @@ def test_end2end_jpl_eph(self):
         """quick functional tests with JPL DE421"""
         #Straight from the C
         cases = ["2015-03-08T19:18:31.786778 479114378970778386 GEI2000 GSE -72417.672749 -51921.225510 -29798.643397 -58063.063795 -73566.510184 -6688.896414",
-                 "2015-03-11T11:58:40.170326 479347187354325578 GSE2000 SM -20164.747998 -52414.140552 -20241.497772 -20870.283837 -46078.983969 -31697.569507",
-                 "2015-03-12T04:25:43.605928 479406410789928239 SM GSM 75645.819112 -40422.353445 -35182.256254 81628.221639 -40422.353445 -17231.208208"]
+                 "2015-03-11T11:58:40.170326 479347187354325578 GSE2000 SM -20164.747998 -52414.140552 -20241.497772 -20868.064112 -46078.015440 -31700.438751",
+                 "2015-03-12T04:25:43.605928 479406410789928239 SM GSM 75645.819112 -40422.353445 -35182.256254 81628.244132 -40422.353445 -17231.101651"]
```

Comment:
Should the C tests also be updated (or are they already?)

Reply:
C tests were already updated.